test/features/page_objects/sdk.rb

```diff
@@ -108,12 +103,13 @@ def modal_button
 end
 
 def open_sdk(driver, config)
+  puts "Open SDK"
   sdk_url = SDK_URL
   config.each do |key, value|
     sdk_url = add_query_to_url(sdk_url, key, value)
   end
-  driver.manage.timeouts.page_load = 30 # ref: https://stackoverflow.com/a/11377772
-  driver.manage.timeouts.implicit_wait = 10 # ref: https://stackoverflow.com/a/11354143
+  driver.manage.timeouts.page_load = 120 # ref: https://stackoverflow.com/a/11377772
+  driver.manage.timeouts.implicit_wait = 30 # ref: https://stackoverflow.com/a/11354143
```

Comment:
Both these timeouts seem very long. Do you really want to wait that long?

Reply:
browser stack is quite slow to boot up (tests take 40min to fully run). So unfortunately yes.